chatmd/local_sync.py

```
@@ -0,0 +1,197 @@
+from __future__ import annotations
+
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Iterable, List, Optional
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)